src/utils/short-type.js

```diff
@@ -1,6 +1,8 @@
 import { shortTypeMapping } from '../config';
 
-const getShortType = (longTypeName, fallback) =>
-  shortTypeMapping[longTypeName] || fallback;
+const getShortType = (name, fallback) => {
+  const longTypeName = name?.split('.').slice(-3).join('.');
```

Comment:
Isn't this basically just doing what your new `get_abbreviated_dataset_type` does? Why don't we just pass the abbreviated dataset type to the frontend instead now we have it in the backend.

Reply:
makes sense. because nowhere in the front end we show the full name.

Reply:
makes sense, i have removed the origin get_dataset_type class and renamed get_dataset_module as get_dataset_type so now everywhere we pass around the shorten dataset name that excludes 'kedro.extras.datasets'